cookbook/workflow/homestead.rst

```diff
@@ -0,0 +1,66 @@
+.. index:: Vagrant, Homestead
+
+Using Symfony with Homestead/Vagrant
+====================================
+
+In order to develop a Symfony application, you might want to use a virtual
+development environment instead of the built-in server or WAMP/LAMP. Homestead_
+is an easy-to-use Vagrant_ box to get a virtual environment up and running
+quickly.
```

Comment:
Would it be appropriate to expand this introduction explaining the advantages and drawbacks of using Homestead with Symfony? For example, this method may run Symfony significantly slower than the native method ... or maybe not, etc.